### ISSUE TITLE
incomplete  upgrade

### ISSUE THREAD

**Issue by dominix** [volitank/nala#31]
Hi everyone, and thank you for this amazing project.

whenever I do a nala upgrade, it often end with not installing a few packages. however if I do an apt dist-upgrade ou full-upgrade every packages install just fine.
could  it be possible to a command that mimic full-upgrade an go further in the resolution of dependencies.

Thank for the good work.

**Comment by volitank** [comment]
Yes, you can use `nala full-upgrade` or `nala upgrade --full`. `dist-upgrade` is also an alias for this.

Please note that in the latest version of Nala there is a slight bug where the two aliases won't update the package list, so you may need to run `nala update` first. `nala upgrade --full` will still update the lists however, and is the preferred choice.

**Comment by volitank** [comment]
This GitHub repository is a mirror and issues are not tracked here.

If this is still an issue, please report it on the canonical GitLab repository:
https://gitlab.com/volian/nala/-/issues

Thanks!
